controllers/administration.js

```diff
@@ -54,50 +54,42 @@ const cutEditOffUrl = (url) => {
 	return workingURL;
 };
 
-const getTableActionsSend = (item, path, state, res) => {
-	const actions = [];
-	if (state === 'submitted' || state === 'closed') {
-		actions.push(
-			{
-				link: path + item._id,
-				class: 'btn-edit',
-				icon: 'edit',
-				title: res.$t('administration.controller.link.editEntry'),
-			},
-			{
-				class: 'disabled',
-				icon: 'archive',
-			},
-			{
-				class: 'disabled',
-				icon: 'paper-plane',
-			},
-		);
-	} else {
-		actions.push(
-			{
-				link: path + item._id,
-				class: 'btn-edit',
-				icon: 'edit',
-				title: res.$t('administration.controller.link.editEntry'),
-			},
-			{
-				link: path + item._id,
-				class: 'btn-disable',
-				icon: 'archive',
-				method: 'delete',
-				title: res.$t('administration.controller.link.completeEntry'),
-			},
-			{
-				link: path + item._id,
-				class: 'btn',
-				icon: 'paper-plane',
-				method: 'post',
-				title: res.$t('administration.controller.link.sendEntryToDevelopmentTeam'),
-			},
-		);
+const getTableActions = (
```

Comment:
Codacy found an issue: ['getTableActions' is assigned a value but never used.](https://app.codacy.com/gh/hpi-schul-cloud/schulcloud-client/pullRequest?prid=6046724)